Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -318,7 +318,7 @@ public protocol NostrEventBuilding {
 
 public extension NostrEvent {
     /// Builder of a ``NostrEvent`` of type `T`.
-    class Builder<T: NostrEvent>: NostrEventBuilding {
+    class Builder<T: NostrEvent>: NostrEventBuilding, LabelBuilding {
```

Comment:
Although labels can exist on any NostrEvent, I've chosen to separate that part of the class out into a separate protocol/extension so that it's easier to maintain, and provides a logical separation.